pydarn/plotting/power.py

```diff
@@ -0,0 +1,146 @@
+#  Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+#  Author: Cooper Ross Robertson, Summer Student 2020
+import matplotlib.pyplot as plt
+import numpy as np
+
+from typing import List
+
+from pydarn import SuperDARNRadars, exceptions, RTP
+
+
+class Power():
+    """
+    Power class to plot SuperDARN RAWACF data power related.
+
+
+    Methods
+    -------
+    plot_lag0
+    """
+    def _str_(self):
+
+        return "This class provides the following methods: \n"\
+                " - plot_interference()"
+
+    @classmethod
+    def plot_lag0(cls, records: List[dict], beam_num: int = 0,
+                  compare: bool = True, frequency: float = 11000,
+                  statistical_calc: object = np.mean):
+
+        """
+        This function will plot lag 0 power as a function of time.
```

Comment:
I know that this is not the RTP class, but I think it's worth being a bit more explicit in this doc to be sure this cannot be confused with an RTP plot. 

```suggestion
        This function will calculate and plot a statistic of the lag-0 
        power of each record as a function of time.
        
        This function applies the statistical function (ex. numpy.mean) to the pwr0 vector
        (lag-0 power for each range) for each record before plotting the results from all 
        records chronologically. 
```